client/src/pages/CreateSurvey/index.jsx

```diff
@@ -49,9 +49,9 @@ const CreateSurvey = ({ history }) => {
 
   useEffect(() => {
     dispatch({ type: 'RESET_CREATE_SURVEY_MODAL_STATE' });
-    dispatch({ type: 'RESET_SURVEY_DATA' });
+    // dispatch({ type: 'RESET_SURVEY_DATA' });
     dispatch({ type: 'RESET_EDIT_SURVEY_STATE' });
-    dispatch({ type: 'RESET_EMPLOYEE_DATA' });
+    // dispatch({ type: 'RESET_EMPLOYEE_DATA' });
```

Comment:
why commented out

Reply:
this state is refreshed before going to the create survey. so that the state does not reset on the actual view /admin/surveys/create if the user refreshes the page